setup.sh

```diff
@@ -0,0 +1,9 @@
+# SPDX-FileCopyrightText: 2023 SUSE LLC
```

Comment:
where is this file used? I don't it as part of the build instructions.

Reply:
it's part of  the REUSE compliance